### DISCUSSION TITLE
Dịch `sequence aware`

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#2752] [Terminology]
Hiện trong #2719 và #2720 em đang dịch từ `sequence aware` là `nhận thức trình tự thời gian`. Cụm từ đầy đủ là `hệ thống gợi ý nhận thức trình tự thời gian`.
Em cũng phân vân với cách dịch `nhận thức tuần tự` hoặc `nhận thức trình tự` cho ngắn gọn.

Trong #2749, @tranyenthy có đề xuất giữ nguyên từ này.
> Mình không chắc là có nên dịch hết ra tiếng Việt không vì nếu dịch hết như vậy việc tra cứu bằng tiếng Anh sẽ hơi bất tiện.

cc: @duythanhvn @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair @tranyenthy

**Comment by cuongvng** [comment]
mình ko có strong opinion, nghiêng về `nhận thức tuần tự` để dễ suy ngược lại `sequence`, vì trước giờ dịch là vậy.

**Reply by lkhphuc** [reply]
Viết đúng phải có dấu gạch giữa `Sequence-aware` để là tính từ ghép, viết tắt cho `aware of sequence`, không phải technical term nên mình thấy nên dịch.
Mình thấy nên dịch là `sequence` là `chuỗi`, không dịch là `tuần tự: sequential` hay thêm `thời gian` vào.
`Sequence-aware recommender system: hệ thống gợi ý có nhận thức về chuỗi`

**Reply by trgiangdo** [reply]
Em thấy cách dịch `nhận thức về chuỗi` của anh @lkhphuc có vẻ ổn.
Em vừa đọc lại thì thấy tác giả định nghĩa
> It is a sequence-aware recommender where the input is an ordered and often timestamped list of past user actions.

`chuỗi` có thể coi là một dãy đã được sắp xếp, và `thời gian` thì chỉ là một trong những cách sắp xếp mà cuốn này xét tới thôi. Cái này do em đọc chưa kĩ.
Nếu mọi người không có ý kiến gì thì em sẽ tạo PR để thêm vào glossary, hoặc em sẽ thực hiện sửa hàng loạt từ này trong các PR hiện có luôn.

**Reply by rootonchair** [reply]
vậy thống nhất dịch là chuỗi nhé

**Comment by rootonchair** [comment]
@trGiang99 sau này bạn mở một PR để rà lại những từ này nhé. Nếu cần, bạn có thể tự mở một issue và tự assign
